addons/full-signature/working.css

```diff
@@ -5,3 +5,9 @@
 ul.activity-stream div {
   width: 90%;
 }
+
+.load-more-wibd {
+  margin-left: 25%;
+  margin-right: 25%;
```

Comment:
Also, can you find a way to center the button properly?

Reply:
It's already better than what other 2.0-styled pages do (`width: auto; margin-left: 45%`).

Reply:
lol really? that's not nice for internationalization.

Reply:
I'd prefer to have the button wrapped in a div so that it can be centered properly tbh